src/features/Nav/Navbar.tsx

```diff
@@ -11,15 +11,9 @@ import Nav from './Nav';
 interface INavbarState {
   loggedIn: boolean;
 }
-interface INavbarProps {
-  showDivider?: boolean;
```

Comment:
I think we might still need this. are we enforcing navbar bottom divider for login pages?

Reply:
@loreina based off the design, it seems like it?

Reply:
okay cool, wasn't sure

Reply:
we can add it back later if anything changes